viewer/packages/sector-loader/src/GltfSectorRepository.ts

```diff
@@ -0,0 +1,132 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+import * as THREE from 'three';
+
+import { ConsumedSector, V9SectorMetadata, WantedSector } from '@reveal/cad-parsers';
+import { BinaryFileProvider } from '@reveal/modeldata-api';
+import { CadMaterialManager } from '@reveal/rendering';
+import { GltfSectorParser, RevealGeometryCollectionType } from '@reveal/sector-parser';
+import { SectorRepository } from '..';
+import { AutoDisposeGroup } from '@reveal/utilities';
+
+export class GltfSectorRepository implements SectorRepository {
+  private readonly _gltfSectorParser: GltfSectorParser;
+  private readonly _sectorFileProvider: BinaryFileProvider;
+  private _materialManager: CadMaterialManager;
+
+  constructor(sectorFileProvider: BinaryFileProvider, materialManager: CadMaterialManager) {
+    this._gltfSectorParser = new GltfSectorParser();
+    this._sectorFileProvider = sectorFileProvider;
+    this._materialManager = materialManager;
+  }
+
+  async loadSector(sector: WantedSector): Promise<ConsumedSector> {
+    const metadata = sector.metadata as V9SectorMetadata;
+
+    if (!metadata.sectorFileName) {
+      return Promise.resolve({
+        modelIdentifier: sector.modelIdentifier,
+        metadata: metadata,
+        levelOfDetail: 2,
```

Comment:
```suggestion
        levelOfDetail: LevelOfDetail.Detailed,
```

Don't hardcode the numeric values.